core-web/libs/edit-content/src/lib/fields/dot-edit-content-file-field/components/dot-form-import-url/dot-form-import-url.component.ts

```diff
@@ -49,9 +56,11 @@ export class DotFormImportUrlComponent implements OnInit {
                 const file = this.store.file();
                 const isDone = this.store.isDone();
 
-                if (file && isDone) {
-                    this.#dialogRef.close(file);
-                }
+                untracked(() => {
+                    if (isDone) {
+                        this.#dialogRef.close(file);
+                    }
+                });
             },
             {
                 allowSignalWrites: true
```

Comment:
`allowSignalWrites` is not necessary, you are not modifying a signal inside the effect.

Reply:
In this case, it is necessary because when we close the dialog, the close action is read by the component that launched the dialog, and in this block, we change the state.

![Screenshot 2024-10-24 at 8 39 33 AM](https://github.com/user-attachments/assets/a5541be9-9c30-49b0-8fc6-d08d3f14daae)

Reply:
oh, I understand. But, this will be a problem in the future. Remember this `allowSignalWrites` will be removed in the future.